perf/grafana-dashboard/dashboard.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: 1
```

Comment:
Why is this file required?

Reply:
This is used for auto creation of dashboards in grafana upon starting the docker container